### PR TITLE
feat(backend): Implemented the rest API for sendMoney

### PR DESCRIPTION
**Pull request by kouamschekina**
Introduced the SendMoneyController to facilitate the initiation of money transfers within our banking application

### CODE REVIEW

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/SendMoneyController.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@RequestMapping("/api/transaction")
+public class SendMoneyController {
+    @Autowired
+    private SendMoney sendMoneyService;
+    public SendMoneyController(SendMoney sendMoneyService){
+        this.sendMoneyService = sendMoneyService;
+    }
+
+    @PostMapping("/transaction")
+    public ResponseEntity<Transaction> sendMoney(@RequestBody Transaction request) {
```
Comment (stephane-segning) [inline power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/SendMoneyController.java:24]: You can skip using `ResponseEntity` and use directly `Transaction`.

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/SendMoneyController.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.adorsys.gis.powerpay.powerpaybackend.domain.Transaction;
+import com.adorsys.gis.powerpay.powerpaybackend.errorhandling.InsufficientFundsException;
+
+@RestController
+@RequestMapping("/api/transaction")
+public class SendMoneyController {
```
Comment (stephane-segning) [inline power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/SendMoneyController.java:16]: You might move this class into a package `controller`.

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work! Please preview these points

**Review by stephane-segning — APPROVED** [review]

**Review by sinke237 — APPROVED** [review]

**Review by NkwaTambe — APPROVED** [review]

**Review by AssahBismarkabah — APPROVED** [review]

**Comment by AssahBismarkabah** [comment]
hello @kouamschekina  the branch has conflicts with the base branch.

**Pull request merged**